infra/vast/README.md

```diff
@@ -0,0 +1,38 @@
+Deployment via (Vast.ai)[vast.ai]
+
+## Local CLI:
+Vast offers a nice UI to deploy instances, and you can use it with the command details below
+```bash
+vastai search offers 'compute_cap > 800 gpu_ram > 20 num_gpus = 1 static_ip=true direct_port_count > 1' 
+# Select the instance and paste it in for <instance-id> to launch
+# Single embedding model deployment:
+vastai create instance <instance-id> --image michaelf34/infinity:latest --env '-p 8000:8000' --disk 40 --args v2 --model-id michaelfeil/bge-small-en-v1.5 --port 8000
+# Multiple models:
+vastai create instance <instance-id> --image michaelf34/infinity:latest --env '-p 8000:8000' --disk 40 --args v2 --model-id mixedbread-ai/mxbai-rerank-xsmall-v1 --model-id  SamLowe/roberta-base-go_emotions --port 8000
+```
+
+## Finding your IP Address:
+Once your instance is up, the ports will be ready. In the Vast UI, you will see a blue button with an IP address and port range: ![vast_instance_view](vast_instance_view.png)
+
+In the view that pops up, you'll see something like: 
+
+```
+Open Ports
+XX.XX.XXX.XX:YYYY ->  8000/tcp
```

Comment:
Is there an option to forward not expose IPv4 and generate a dynamic adress. `http://some-random-name.ports.vast.ai`? @aowen14